src/themes/OLH/templates/journal/article.html

```diff
@@ -161,7 +161,7 @@ <h4 class="number">
                                                 </div>
                                                 <div class="large-4 columns">
                                                     <h4 class="number">
-                                                        {{ article.metrics.alm.total }}<span>{% trans "Alt Metrics" %}</span>
+                                                        {{ article.citations|length }}<span>{% trans "Citations" %}</span>
```

Comment:
another question: Does this work? docs suggest that it can handle lists and strings,plus IIRC querysets don't implement `__len__`, you have to call queryset.count() instead.

Reply:
Yes it does work. This filter wraps len() around the passed var. Could change to count tho.

Reply:
> Yes it does work. This filter wraps len() around the passed var. Could change to count tho.

That's what I meant, Queryset doesn't implement '__len__' so `len()` won't work...

Reply:
QuerySets do.implement len. Anyway, swapping to .count() is recommended.

Reply:
Oh I see, `len()` queries all the rows and then counts them instead of issuing a count query to the DB. Yeah, worth swapping for a `count()` still